### PR TITLE
[13.0][ADD] maintenance_plan_employee: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Locked by:
- [x] `maintenance_request_employee`: https://github.com/OCA/maintenance/pull/302

@Tecnativa TT41735

### CODE REVIEW

**Comment by victoralmau** [comment]
Please @pedrobaeza and @CarlosRoca13 can you review it?

**Review by CarlosRoca13 — APPROVED** [review]
Code review

File: maintenance_plan_employee/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/maintenance-13-0/maintenance-13-0-maintenance_plan_employee
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/240/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds the option to define employees in maintenance plans.
```
Comment (pedrobaeza) [inline maintenance_plan_employee/README.rst]:
```suggestion
This module adds the option to define employees in the maintenance plans.
```

File: maintenance_plan/tests/common.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as test_common
+from odoo import fields
+
+
+class TestMaintenancePlanBase(test_common.TransactionCase):
```
Comment (pedrobaeza) [inline maintenance_plan/tests/common.py]: Do it `SavepointCase` now that you change it.

File: maintenance_plan/tests/common.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as test_common
+from odoo import fields
+
+
+class TestMaintenancePlanBase(test_common.SavepointCase):
+    def setUp(self):
+        super().setUp()
```
Comment (pedrobaeza) [inline maintenance_plan/tests/common.py]: But you should also convert this to `setUpClass`.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**